test/fiddle/test_closure.rb

```diff
@@ -149,5 +149,14 @@ def test_memsize_ruby_dev_42480
         end
       end
     end
+
+    if defined?(Ractor)
```

Comment:
Could you use `ractor` instead like https://github.com/ruby/pathname/pull/38 ?
We need to run tests that use Ractor after tests that don't use Ractor. It's for running tests that don't use Ractor in single-ractor mode not multi-ractor mode.

Reply:
These tests just use `make_shareable` and `shareable?`, and don’t put it in multi-ractor mode, I think.

Reply:
Ah, you're right.
We don't need to use `ractor`.

How about adding `omit("Need Ractor support") unless defined?(Ractor) to `assert_ractor_shareable`?